### PR TITLE
fix: Update copyright to 2022 for newly generated files.

### PR DESCRIPTION
**Pull request by blakeli0**

### CODE REVIEW

File: src/main/java/com/google/api/generator/gapic/composer/comment/CommentComposer.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 public class CommentComposer {
   private static final String APACHE_LICENSE_STRING =
-      "Copyright 2021 Google LLC\n\n"
+      "Copyright 2022 Google LLC\n\n"
```
Comment (meltsufin) [inline src/main/java/com/google/api/generator/gapic/composer/comment/CommentComposer.java:23]: Can we just put in the current year, so we don't have to update it every year?

File: BUILD.bazel [hunk 2]
```diff
@@ -209,7 +209,6 @@ GOLDEN_UPDATING_UNIT_TESTS = [
     args = [test_class],
     runtime_deps = [
         ":gapic_generator_java",
-        ":gapic_generator_java-test",
```
Comment (meltsufin) [inline BUILD.bazel:212]: why this change?

**Review by meltsufin — CHANGES_REQUESTED** [review]

File: BUILD.bazel (hunk 2 above)
Comment (blakeli0) [reply BUILD.bazel:212]: `gapic_generator_java-test` does not exist, `bazel test //...` would error out without this change.

File: src/main/java/com/google/api/generator/gapic/composer/comment/CommentComposer.java (hunk 1 above)
Comment (blakeli0) [reply src/main/java/com/google/api/generator/gapic/composer/comment/CommentComposer.java:23]: I thought about it, the concern is that our golden tests would start failing all of sudden in the beginning of next year, which is not a good behavior in my opinion. I can update it to use current year if this behavior is OK.

**Review by meltsufin — APPROVED** [review]
